Upgrade K8s to v1.22 (#2792)
epiphany-platform/epiphany: docs/home/howto/KUBERNETES.md

If you need your GUI accessible outside, you would have to change your firewall rules.

GUI should be reachable at: https://localhost:30104/auth

## Audit logs

Audit logs are stored in `/var/log/kubernetes/audit/` directory on control plane nodes.
There is a possibility to configure a rotation:

```yaml
kind: configuration/kubernetes-master
title: Kubernetes Master Config
name: default
specification:
  advanced:
    api_server_args:
      audit-log-maxbackup: 10
      audit-log-maxsize: 200
```

Refer to K8s [documentation](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#log-backend) to check the

> meaning of these values. Default values are specified in the sample above.

Reviewer: ```suggestion
meaning of these values. The sample above shows the defaults.
```
